app/Support/MailConfiguration/Drivers/PostmarkConfigurationDriver.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Support\MailConfiguration\Drivers;
+
+use Illuminate\Contracts\Config\Repository;
+
+class PostmarkConfigurationDriver extends MailConfigurationDriver
+{
+    public function name(): string
+    {
+        return 'postmark';
+    }
+
+    public function validationRules(): array
+    {
+        return [
+            'default_from_mail' => 'required',
+            'postmark_mails_per_second' => 'required|numeric|between:1,100',
+            'postmark_token' => 'required',
+            'postmark_user' => 'required',
```

Comment:
How does Mailcoach know everything else about Postmark SMTP, is this information known in a lowre level library that is used by Laravel?

Reply:
Yes, by default there is a package used with some Postmark specific things.